### DISCUSSION TITLE
Shared traceId with Sleuth and skywalking

### DISCUSSION THREAD

**Discussion by Toxa45** [apache/skywalking#10676] [Q&A] [answered]
Good afternoon.
Does the Skywalking trace ID support customization and can it be associated with the Spring Cloud Sleuth trace ID.

**Comment by wu-sheng** [comment, marked as answer]
No, we wouldn't. Because auto-instrumentation always creates tracing context(including your trace ID) before Sleuth does anything.
So, we can't afford to reset the trace ID after the tracing had began.
